tests/python/test_mi_bloom_filter.py

```diff
@@ -25,6 +25,7 @@ def setUp(self):
     def set_up_mi_bf_1(self):
         
         self.mi_bf_1 = btllib.MIBloomFilter8(1024 * 1024, 3, "ntHash")
+        self.mi_bf_1.set_k() = 10
```

Comment:
I think this should be `self.mi_bf_1.set_k(10)`?

Reply:
thanks for spotting that. That is strange, the test didn't fail locally. Maybe I didn't save before running